### PR TITLE
[improvement](log) optimize template function log for performance

### PR DESCRIPTION
**Pull request by xiaokang**
## Proposed changes

change log level to debug and use format in template function log for performance.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yiguolei** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java [hunk 1]
```diff
@@ -345,10 +345,10 @@ private Function getFunction(Function desc, Function.CompareMode mode, List<Func
     public Function specializeTemplateFunction(Function templateFunction, Function requestFunction, boolean isVariadic) {
         try {
             boolean hasTemplateType = false;
-            LOG.debug("templateFunction signature: " + templateFunction.signatureString()
-                        + "  return: " + templateFunction.getReturnType());
-            LOG.debug("requestFunction signature: " + requestFunction.signatureString()
-                        + "  return: " + requestFunction.getReturnType());
+            LOG.debug("templateFunction signature: {}, return type: {}",
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java:348]: if (Log.isdebugenabled()) {
xxxx
}

File: fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java [hunk 2]
```diff
@@ -397,12 +397,12 @@ public Function specializeTemplateFunction(Function templateFunction, Function r
                         specializedFunction.getReturnType().specializeTemplateType(
                         requestFunction.getReturnType(), specializedTypeMap, true));
             }
-            LOG.debug("specializedFunction signature: " + specializedFunction.signatureString()
-                        + "  return: " + specializedFunction.getReturnType());
+            LOG.debug("specializedFunction signature: {}, return type: {}",
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java:400]: if (Log.isdebugenabled()) {
xxxx
}

File: fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java [hunk 3]
```diff
@@ -397,12 +397,12 @@ public Function specializeTemplateFunction(Function templateFunction, Function r
...
+                        specializedFunction.signatureString(), specializedFunction.getReturnType());
             return hasTemplateType ? specializedFunction : templateFunction;
         } catch (TypeException e) {
             if (inited) {
-                LOG.warn("specializeTemplateFunction exception", e);
+                LOG.debug("specializeTemplateFunction exception", e);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java:405]: if (Log.isdebugenabled()) {
xxxx
}

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by xiaokang** [comment]
run buildall

**Comment by yiguolei** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.59 seconds
 stream load tsv:          544 seconds loaded 74807831229 Bytes, about 131 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17162197181 Bytes

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 47.56 seconds
 stream load tsv:          551 seconds loaded 74807831229 Bytes, about 129 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.1 seconds inserted 10000000 Rows, about 343K ops/s
 storage size: 17162330294 Bytes

**Review by morrySnow — APPROVED** [review]

**Pull request merged**